Funda mapjes - kopie/styles/nieuwefundastyle.css

```
@@ -0,0 +1,342 @@
+:root{
+    --primaire-color: #F7A100;
+    --secundaire-color: #0071B3;
+    --primaire-lettertype: Proxima Nova;
+    --navbarlettertype: 15px;
+}
+
+.maandlasten {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: proxima nova, sans-serif;
+}
+
+/* BEGIN HEADER */
+
+nav {
+    display: flex;
+    justify-content: space-between;
+    /* width: 414px; */
+    height: 60px;
+    border: 0px;
+    top: 0px;    
+    background-color: var(--primaire-color);
+}
+
+.navbar-left {
+    display: flex;
+    justify-content: space-between;
+}
+
+.button-menu {
+    height: 100% ;
+    background-color: var(--primaire-color);
+    border-color: var(--primaire-color);
+    border: 0px;
+    color: white;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    font-size: 12px;
+    margin-left: 15px;
+    width: 50px;
+}
+
+.logo{
+    position: relative;
+    width: 100px;
+    height: 75px;
+}
+
+.button-house {
+    display: flex;
+    width: 70px;
+    height: 100%;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    background-color: var(--primaire-color);
+    border: 0px;
+    color: white;
+    text-decoration: none;
+    font-size: 12px;
+}
+
+.my-house{
+    display: flex;
+}
+
+.alog{
```

Zorg voor een consistentie indentation in alle bestanden in het project. Je kunt dit eenvoudig oplossen door een extensie te gebruik in Visual Studio Code (als je deze gebruikt). Je kan het zo instellen dat wanneer je een bestand opslaat na het maken van een wijziging, de editor automatisch de indentation fixed. Dit zou je moeten automatiseren, zodat je er niet over na hoeft te denken. 
```suggestion
.alog {
```